doc/source/user/gdal_python_utilities.rst

```diff
@@ -0,0 +1,62 @@
+.. _gdal_python_utilities:
+
+================================================================================
+GDAL Python Utilities
+================================================================================
+
+The GDAL python utilities are included with GDAL. If you've installed
+GDAL you already have them. However you may want to use a newer or older
+version of the utilities without changing GDAL. This is where
+**gdal-utils** comes in.
+
+**gdal-utils**: is the GDAL Python Utilities *distribution*. This is
+what you install. It's home page is https://pypi.org/project/gdal-utils/
+. Install it with ``pip install gdal-utils`` and it will up or downgrade
```

Comment:
The "it will up or downgrade ..." part. is a bit mysterious on reading  Is it just refering to pip standard behavior ? However what could be mentioned is the interaction with the "gdal" pip package.  gdal-utils and gdal pip packages both provide the GDAL utilities so depending in the order they are installed, overwriting could occur. as mentionned in the RFC

Reply:
Yeah I struggle with this sentence. The goal is trying to communicate that "whichever is last most recenty installed wins" without adding as many sentences as in the RFC. Maybe it's best just to leave it as simply _"Install with ``pip install gdal-utils``"_ as the concept of up or downgrading is already introduced in the paragraph above.